### PR TITLE
couple minor updates for M1 support

### PR DESCRIPTION
**Pull request by fundthmcalculus**

### CODE REVIEW

File: ruby/Gemfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-source "https://rubygems.org"
-gemspec
\ No newline at end of file
+source 'https://rubygems.org'
```
Comment (allibell) [inline ruby/Gemfile:1]: paste (2)

File: python/tests/test_trinsic_services.py [hunk 2]
```diff
@@ -15,6 +16,9 @@
 # Due to some issues with python and async io test cases, we have to run each sample in a separate asyncio event loop.
 
 class TestServices(unittest.TestCase):
+    def test_python_platform(self):
+        print(f"Running on: {platform.platform()}")
```
Comment (allibell) [inline python/tests/test_trinsic_services.py:20]: should we have any asserts here?  like non-null or something?  or is this just for debugging utility

**Review by allibell — COMMENTED** [review]
genuinely curious about how double quotes might relate to M1 Ruby things, otherwise LGTM!

File: ruby/Gemfile (hunk 1 above)
Comment (fundthmcalculus) [reply ruby/Gemfile:1]: Rubocop style guidelines. Single quotes in Ruby make it a string literal, double quotes allow for string interpolation and stuff like that. This makes it clear it's just a string of characters, nothing exciting.

File: python/tests/test_trinsic_services.py (hunk 2 above)
Comment (fundthmcalculus) [reply python/tests/test_trinsic_services.py:20]: No, this is a debug test (that I plan to replicate everywhere) that will enable us to ensure what platform the code *thinks* it's running on, eg for M1 (native) vs M1 (emulated intel).
If it makes you feel better, I can `assert platform.platform() is not None`. 😁

Comment (allibell) [reply python/tests/test_trinsic_services.py:20]: Nah - just wanted to check you didn’t intend to have legit unit test stuff here and then lose some logic in the shuffle.  That makes sense!

**Review by allibell — APPROVED** [review]

**Pull request merged**